src/nncf/torch/function_hook/hook_executor_mode.py

```diff
@@ -294,23 +273,16 @@ def get_current_executed_op_name(self, fn_name: str) -> str:
         """
         Get the name of the current operation being executed.
 
+        Note: should be called only ones because it updates op_calls.
+
         :param fn_name: The function name of the operation.
         :return: The name of the operation.
         """
         module_name = self.get_current_relative_name()
         op_name = generate_normalized_op_name(module_name, fn_name)
         call_id = self.op_calls[op_name]
-        return generate_normalized_op_name(module_name, fn_name, call_id)
-
-    def register_op(self, fn_name: str) -> None:
-        """
-        Register an operation call for the current module and increment call counter.
-
-        :param fn_name: The function name of the operation.
-        """
-        module_name = self.get_current_relative_name()
-        op_name = generate_normalized_op_name(module_name, fn_name)
         self.op_calls[op_name] += 1
```

Comment:
Can it be confusing that just getting name via `get_current_executed_op_name` increase counter of calls? 
I'd increase the counter explicitly outside of this function or keep it inside `register_op`.

Reply:
Renamed function to 
`get_next_op_call_name` and update docstring to be clear